config/locales/client.en.yml

```diff
@@ -2,6 +2,8 @@ en:
   js:
     audio_message:
       modal:
-        title: "Click to record"
+        title-start: "Click to Start Recording"
+        title-stop: "Click to Stop Recording"
+        title-review: "Click to Review Recording"
```

Comment:
I was trying to keep the text very brief and short, and only add text when it's not self-explanatory. I feel like the "Click the Review Recording" is a bit elaborate for a simple audio player. The "Click to Stop Recording" is also a lot of text that enlarges the modal view and draws attention when the stop button seems to be fairly straight forward? What do you think?

I kind of liked the simplicity and graphical emphasis on the buttons, which makes it self-explanatory for non-english speakers as well.